stac_model/output.py

```diff
@@ -26,16 +32,13 @@ class ResultArray(BaseModel):
         pattern="^(uint8|uint16|uint32|uint64|int8|int16|int32|int64|float16|float32|float64)$",
     )
 
-class ClassObject(BaseModel):
-    value: int
-    name: str
-    description: Optional[str] = None
-    title: Optional[str] = None
-    color_hint: Optional[str] = None
-    nodata: Optional[bool] = False
+MLMClassification = Annotated[Classification,
+                             PlainSerializer(lambda x: x.to_dict(), return_type=Dict[str, Any], when_used='json')]
```

Comment:
Replacing the Annotated type with this class doesn't work unfortunately

```python
/home/rave/work/dlm-extension/tests/test_schema.py::test_model_metadata_to_dict failed with error: Test failed with exception
tests/test_schema.py:8: in mlmodel_metadata_item
    model_metadata_stac_item = eurosat_resnet()
stac_model/examples.py:107: in eurosat_resnet
    class_objects = [
stac_model/examples.py:108: in <listcomp>
    MLMClassification.create(value=class_map[class_name], description=class_name, name=class_name)
../../.cache/pypoetry/virtualenvs/stac-model-YQXQDXJF-py3.10/lib/python3.10/site-packages/pystac/extensions/classification.py:112: in create
    c = cls({})
E   TypeError: BaseModel.__init__() takes 1 positional argument but 2 were given
```

I suggest for now we stick with this since it works, we can always change the underlying implementation in the future if it is confusing for contributors.